ugs-core/src/com/willwinder/universalgcodesender/pendantui/PendantConfigBean.java

```diff
@@ -55,6 +59,41 @@ public String toString() {
 			return "StepSizeOption [value=" + value + ", label=" + label + ", selected=" + selected + "]";
 		}
 	}
+
+	public static class UnitAmtOption{
```

Comment:
I'm not totally clear on the purpose of this, would it be possible to cache just the current Unit instead of a list?

Reply:
Nevermind, I see that this is the pattern used by the other settings as well.